src/Modules/Backend/tests/BackendWebTestCase.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\tests;
```

Comment:
Why not `ForkCMS\Modules\Backend\Tests`, tests with uppercase?

Reply:
I based the naming of the directory on https://symfony.com/doc/current/bundles/best_practices.html#tests.
It does look a bit weird, but it is in lowercase because the directory name is lowercase.
We could deviate here and rename the tests directory to Tests and then we could have it in upper case?

Reply:
No, follow the best practices.